.travis.yml

```diff
@@ -0,0 +1,18 @@
+language: node_js
+
+cache: npm
+
+node_js:
+  - "6"
+  - "8"
+  - "10"
+
+branches:
+  only:
+    - master
+
+notifications:
+  email: false
+
+before_install:
+  - npm config set //registry.npmjs.org/:_authToken=$NPM_TOKEN
```

Comment:
Reminder to add this env var when you set up the repo on Travis.

Reply:
yep. it was done